css/gallery.css

```diff
@@ -0,0 +1,149 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit; 
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%; 
+}
+
+body {
+    max-height: 100vh;
+    max-width: 100%;
+    background: #1a2036;
+    font-family:'Times New Roman', Times, serif;
+  }
+
+.container {
+  width: 60%;
+  margin: 2rem auto; 
+}
+.home {
+    display: flex;
+    flex-direction: row;
+    gap: 4rem;
+    margin: 4rem 2rem;
+    max-height: 100vh;
+  }
+  
+  nav {
+    height: 80vh;
+    width: 5rem;
+    background-color: #a50000;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    border: 0px solid #1A2036;
+    border-radius: 1rem;
+  
+  }
+  
+  nav a {
+    width: 3rem;
+    display: flex;
+    align-items: center;
+    border: 0px solid;
+    border-radius: 0.5rem;
+  }
+  
+  nav a:hover {
+    background-color: #fff;
+  }
+  
+  nav a:hover svg {
+    color: #999;
+  }
+  
```

Comment:
suggestion for active page representation:

nav a.active{
    background: #666;
}

Reply:
resolved